### ISSUE TITLE
an error in example.ipynb

### ISSUE THREAD

**Issue by kangkang59812** [lukemelas/EfficientNet-PyTorch#6]
`preds = torch.topk(logits, k=5).indices.squeeze(0).tolist()`

 I think you want to find the indexs for top5 using indices , but tuple has no attributes indices when I using the code above.

`preds = torch.topk(logits, k=5)[1].squeeze(0).tolist()`
It works well.

**Comment by lukemelas** [comment]
What version of PyTorch are you using? 

The first line of code works well for me (PyTorch 1.1).

**Comment by cardoso-neto** [comment]
```bash
pip freeze > requirements.txt
```
If using conda envs:
```bash
conda env export > environment.yaml
```
@lukemelas
Could you please run one of these commands and upload the file for us?
This will help avoiding this sort of issue in the future.

**Comment by lukemelas** [comment]
Yes, I'll update with a `requirements.txt` file in the next commit.

**Comment by kangkang59812** [comment]
> What version of PyTorch are you using?
> 
> The first line of code works well for me (PyTorch 1.1).

In 1.0.1, it returns a tuple. While in 1.1 , the result has the attr 'indices'.
